<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Image Toolbox</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuFlip">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Flip</string>
     </property>
     <addaction name="actionFlip_Horizontally"/>
     <addaction name="actionFlip_Vertically"/>
    </widget>
    <widget class="QMenu" name="menuGenerate">
     <property name="title">
      <string>Generate</string>
     </property>
     <addaction name="actionAdd_Noise"/>
     <addaction name="actionRandomize_Colors"/>
     <addaction name="actionRandom_Pixel_Image"/>
     <addaction name="actionSine_Image"/>
     <addaction name="actionSingle_Color_Image"/>
    </widget>
    <widget class="QMenu" name="menuTransforms">
     <property name="title">
      <string>Transforms</string>
     </property>
     <addaction name="actionRotate"/>
    </widget>
    <addaction name="actionBinarize"/>
    <addaction name="actionAdjust_Colors"/>
    <addaction name="actionBlend"/>
    <addaction name="actionConvert"/>
    <addaction name="actionConvolve"/>
    <addaction name="menuFlip"/>
    <addaction name="menuGenerate"/>
    <addaction name="actionGrayscale"/>
    <addaction name="actionNegative"/>
    <addaction name="actionNormalize"/>
    <addaction name="actionPixelate"/>
    <addaction name="actionResize"/>
    <addaction name="actionSepia"/>
    <addaction name="menuTransforms"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFit_to_Window"/>
    <addaction name="actionOriginal_Size"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_Help"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>Statistics</string>
    </property>
    <addaction name="actionHistogram"/>
    <addaction name="actionPixel_Count"/>
    <addaction name="actionUnique_Colors"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuStatistics"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen_Camera">
   <property name="text">
    <string>Open Camera</string>
   </property>
  </action>
  <action name="actionClose_Camera">
   <property name="text">
    <string>Close Camera</string>
   </property>
  </action>
  <action name="actionFit_to_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit to Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionNegative">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionSepia">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sepia</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionBinarize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Binarize</string>
   </property>
  </action>
  <action name="actionFlip_Horizontally">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Horizontally</string>
   </property>
  </action>
  <action name="actionFlip_Vertically">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Vertically</string>
   </property>
  </action>
  <action name="actionRandom_Pixel_Image">
   <property name="text">
    <string>Random Pixel Image</string>
   </property>
  </action>
  <action name="actionSine_Image">
   <property name="text">
    <string>Sine Image</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
   <property name="toolTip">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="text">
    <string>Convert</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionOriginal_Size">
   <property name="text">
    <string>Original Size</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionShow_Help">
   <property name="text">
    <string>Show Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUnique_Colors">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Unique Colors</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionBlend">
   <property name="text">
    <string>Blend</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAdd_Noise">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Noise</string>
   </property>
  </action>
  <action name="actionPixel_Count">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pixel Count</string>
   </property>
  </action>
  <action name="actionNormalize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normalize</string>
   </property>
  </action>
  <action name="actionAdjust_Colors">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Adjust Colors</string>
   </property>
  </action>
  <action name="actionConvolve">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Convolve</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSingle_Color_Image">
   <property name="text">
    <string>Single Color Image</string>
   </property>
  </action>
  <action name="actionTake_Picture">
   <property name="text">
    <string>Take Picture</string>
   </property>
  </action>
  <action name="actionRandomize_Colors">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Randomize Colors</string>
   </property>
  </action>
  <action name="actionPixelate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pixelate</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
